### PR TITLE
Move constants to Constants file

### PR DESCRIPTION
**Pull request by dfangshuo**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

[//]: # "Describe what's new in this PR"

Move constants to constants file

### How to Review

[//]: # 'The order in which to review files and what to expect when testing locally'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc"

### Next steps

[//]: # "What doesn't work yet, what's NOT in this PR/has to be done "

### Tests Performed, Edge Cases

[//]: # 'If you made changes/additions to `airtable.js` make sure you update/add'
[//]: # '[at least 1] corresponding unit tests in `airtable.spec.js`'

### Screenshots

[//]: # "Add screenshots!!! If you'd like"

CC: @dfangshuo

### CODE REVIEW

File: Constants.js [hunk 1]
```diff
@@ -1,4 +1,11 @@
 export default {
   ROOT_URL: 'https://peoplepower-node.herokuapp.com',
-  PDF_DELETE_DELAY: 20
+  BILLS_BASE_URL: 'https://utilityapi.com/api/v2/bills',
+  EBCE_REBATE_STRING: 'Credited to (Debited from) NEM Balance',
```
Comment (aivantg) [inline Constants.js]: I left the above two constants where they were because they were rather specific features of how the utilityAPI code functioned and it felt like it actually be MORE inconvenient to store them here than in the same file when debugging. 

Just an opinion really, up to you! I can add back in my new PR

Comment (aivantg) [reply Constants.js]: I suppose this is coming from a view where the constants file are like tweak-able aspects of the server. These are things we likely wouldn't tweak without editing the code too.

File: utils/utilityApi.js [hunk 2]
```diff
@@ -6,16 +6,14 @@ A set of low-level functions that gives us the details we need from the Utility
...
 import fetch from 'node-fetch';
 import moment from 'moment';
+import Constants from '../Constants';
 
-const API_KEY = process.env.UTILITY_API_KEY;
-const EBCERebateString = 'Credited to (Debited from) NEM Balance';
-
-// Base URL for Utility API BILLS
-const billsBaseUrl = 'https://utilityapi.com/api/v2/bills';
+const meterURL = `?meters=${meterId}`;
```
Comment (aivantg) [inline utils/utilityApi.js]: since this uses `meterId`, it can't be outside the function

File: utils/email.js [hunk 3]
```diff
@@ -39,9 +38,8 @@ const sendEmail = async email => {
 };
 
 const sendInviteEmail = async RECORD_ID => {
-  const baseUrl = 'https://peoplepower.netlify.com/onboarding';
-  const inviteParameter = `?token=${RECORD_ID}`;
-  const inviteLink = baseUrl + inviteParameter;
+  const inviteParameter = `?${Constants.INVITE_BASE_URL}=${RECORD_ID}`;
```
Comment (aivantg) [inline utils/email.js]: I think you meant `Constants.INVITE_PARAM

**Comment by aivantg** [comment]
Closing in favor of #7
